### PR TITLE
add default functions for jump forward/back interval

### PR DESCRIPTION
**Pull request by simonista**

### CODE REVIEW

**Comment by benternet** [comment]
@simonista we have a history of updating the build files in the PR that contains the src file changes as well. mind updating with those?

File: src/js/mep-player.js [hunk 1]
```diff
@@ -52,6 +52,13 @@ function constrainedSeekTo(player, media, targetTime) {
 		defaultSeekForwardInterval: function(media) {
 			return (media.duration * 0.05);
 		},
+		defaultJumpBackwardInterval: function(media) {
+			return (media.duration * 0.05);
+		},
+		// default amount to move forward when forward key is pressed
```
Comment (mziwisky) [inline src/js/mep-player.js]: this comment should differentiate the option from that of `defaultSeekForwardInterval`, and `defaultJumpBackwardInterval` should have a corresponding comment

**Review by mziwisky — CHANGES_REQUESTED** [review]
Nice catch, just asking for a little more in the code comments

**Review by a-b-r-o-w-n — CHANGES_REQUESTED** [review]
The code looks good. Could you update the commit message to include 'interval'? The message now makes it seem like the functions that were added were for jumping, not for the interval.

**Comment by simonista** [comment]
@benternet / @mziwisky / @astephenb : done

File: build/mediaelement-and-player.js [hunk 2]
```diff
@@ -2573,6 +2573,14 @@ function constrainedSeekTo(player, media, targetTime) {
 		defaultSeekForwardInterval: function(media) {
 			return (media.duration * 0.05);
 		},
+    // default amount to move back when Page Up key is pressed
+		defaultJumpBackwardInterval: function(media) {
+			return (media.duration * 0.05);
+		},
+    // default amount to move back when Page Down key is pressed
```
Comment (mziwisky) [inline build/mediaelement-and-player.js]: copy-pasta fail -- s/back/forward.  (also s/  /\t/g)

**Review by mziwisky — CHANGES_REQUESTED** [review]

File: build/mediaelement-and-player.js (hunk 2 above)
Comment (simonista) [reply build/mediaelement-and-player.js]: 🤦‍♂️ thanks

**Review by mziwisky — APPROVED** [review]
👍

**Comment by simonista** [comment]
@astephenb could you re-review this to unblock it?

**Review by a-b-r-o-w-n — APPROVED** [review]

**Comment by simonista** [comment]
@benternet any chance we could merge this today, and then I can re-build generated files for the other PR?

**Comment by benternet** [comment]
ah sorry @simonista didn't see this earlier today. we typically have the author merge upon review complete and QA complete. do you happen to have QA resources that would test this or should we wait on Arc QA resources? our QA team is pretty slammed ATM but i'll check with them (@kyler-instructure) tomorrow.

**Comment by simonista** [comment]
Hey @benternet, I could show my team's QA person locally that without this there were console errors and with it there aren't. Since it's just adding to default config, I'm not sure how valuable regression testing is.  What do you think @kyler-instructure, what would you like to see here?

**Review by kyler-instructure — APPROVED** [review]
QA +1

**Pull request merged**